scalableminds/webknossos
# Check user setting boundaries in reducer to avoid invalid values

Before, boundaries needed to be checked everywhere an `updateUserSettingAction` for a numeric value was dispatched. Forgetting the bounds check did lead to invalid values. Now the reducer takes care of that.
"Funnily" enough, we actually had a test, setting an invalid move value, which should have failed but did not :see_no_evil: I added a regression test.

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- Change brush size (using Shift + Scroll and Shift + I/O), change move value (using G/H), change particle size in flight mode (using Shift + Scroll), change scale (using K/L).
- The respective boundaries should be respected and reloading wK should not lead to errors.

### Issues:
- fixes https://discuss.webknossos.org/t/tracing-error-message/1404

------
